PowerShellBuild/IB.tasks.ps1

```diff
@@ -113,7 +115,7 @@ $genMarkdownPreReqs = {
 }
 
 # Synopsis: Generates PlatyPS markdown files from module help
-task GenerateMarkdown StageFiles, {
+task GenerateMarkdown -if ($genMarkdownPreReqs) StageFiles,{
```

Comment:
@devblackops minor bug here I think, this should be `(. $genMarkdownPrereqs)`, my bad, may want to do a quick patch commit since you already merged the PR.

Reply:
OK. I'll fix and publish v0.6.1